.circleci/config.yml

```
@@ -35,6 +35,55 @@ references:
           POSTGRES_PASSWORD: mirror_node_pass
           POSTGRES_USER: mirror_node
 
+commands:
+  gcloud_image_untag:
+    description: "Untags old master images"
+    parameters:
+      imagepath:
+        type: string
+        default: "importer"
+    steps:
+      - run:
+          name: "Untags old master images"
+          command: |
+            set -x
```

Should be set -ex to fail fast